### ISSUE TITLE
Canonical url missing domain name

### ISSUE THREAD

**Issue by dumbflex** [nephila/djangocms-blog#166]
Using django=1.7.10, djangocms=3.1.3, djangocms-blog=0.6.2

I'm missing the domain name in the canonical url.

Seems like in the templates the {{ SITE.domain }} doesn't work in the canonical url line.

**Comment by yakky** [comment]
How do you generate canonical URL? ``django-meta-mixin`` way to generate url is the ``Post.get_full_url`` method (which uses the same logic of Django's ``build_absolute_uri``)

**Comment by dumbflex** [comment]
I added a {% block canonical_url %}{% endblock canonical_url %} to my base template since it's defined in post_detail.html template line 6

```
{% block canonical_url %}<link rel="canonical" href="{{ SITE.domain }}{{ view.get_view_url }}"/>{% endblock canonical_url %}
```

but it seems that {{ SITE.domain }} is empty.

**Comment by yakky** [comment]
You can use ``meta`` context variable to create the canonical URL: ``<link rel="canonical" href="{{ meta.url }}"/>``
``{{ SITE }}`` is anyway handled by Django, not by the blog application

**Comment by dumbflex** [comment]
Ok that works.

Would it be better to use that in the djangocms-blog templates?

**Comment by yakky** [comment]
@dumbflex I totally overlooked that djangocms-blog ships the canonical URL in the template!
Going to fix that, thanks for reporting!

**Comment by yakky** [comment]
@dumbflex fixed in #169 fi you want to test it
